resources/namespace.rb

```diff
@@ -23,24 +23,22 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-resource_name :sensu_organization
+resource_name :sensu_namespace
 
 action_class do
   include SensuCookbook::Helpers
 end
 
 property :config_home, String, default: '/etc/sensu'
 
-property :description, String
```

Comment:
Hmm I see its not in the spec but I think namespaces should have descriptions rather than relying on super long descriptive names. @portertech @grepory thoughts?

Reply:
Again, maybe true and worth an interesting conversation about design, but based on current master spec (assuming I'm looking in the right place), name is all it supports right now: https://github.com/sensu/sensu-go/blob/master/api/core/v2/namespace.proto so I'm leaning towards at least feature parity in this PR. 

Input from @portertech or @grepory is always welcome.

Reply:
We did not add a description to namespaces. The API has been frozen.

Reply:
This sounds like intended behavior @majormoses ? I don't think the cookbook should impose design constraints so we'll have to just work with the current frozen API.